packages/@webex/plugin-meetings/src/common/errors/webex-errors.ts

```diff
@@ -107,6 +107,27 @@ class UserInLobbyError extends WebexMeetingsError {
 export {UserInLobbyError};
 WebExMeetingsErrors[UserInLobbyError.CODE] = UserInLobbyError;
 
+/**
+ * @class SpaceIDDeprecatedError
+ * @classdesc Raised whenever the user passes Space ID as destination for create meeting.
+ * @extends WebexMeetingsError
+ * @property {number} code - 101
```

Comment:
On what basis do we decide these code numbers?

Reply:
We randomly decided on this error code, and wanted to do the same error mapping as https://docs.agora.io/en/3.x/voice-calling/reference/error-codes 
@arun3528

Reply:
Alright. Makes sense. We need to write a documentation for these errors. Here's the Spark ID for the same: SPARK-465209